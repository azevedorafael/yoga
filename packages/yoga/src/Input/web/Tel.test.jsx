import React from 'react';
import { render } from '@testing-library/react';

import { ThemeProvider, Input } from '../..';

describe('<Input.Tel />', () => {
  describe('Snapshots', () => {
    it('should match with default input', () => {
      const { container } = render(
        <ThemeProvider>
          <Input.Tel />
        </ThemeProvider>,
      );

      expect(container).toMatchSnapshot();
    });
  });
});
