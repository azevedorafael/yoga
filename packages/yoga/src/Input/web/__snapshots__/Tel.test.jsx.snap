// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input.Tel /> Snapshots should match with default input 1`] = `
.c4 {
  position: absolute;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  pointer-events: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  top: 24px;
  left: 16px;
  background-color: #FFFFFF;
  font-size: 14px;
  font-weight: 400;
  color: #9898A6;
  -webkit-transform: translateY(-40%);
  -ms-transform: translateY(-40%);
  transform: translateY(-40%);
  -webkit-transition-duration: 500ms;
  transition-duration: 500ms;
  -webkit-transition-timing-function: cubic-bezier(0,0.75,0.1,1);
  transition-timing-function: cubic-bezier(0,0.75,0.1,1);
}

.c2 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background-color: transparent;
  outline: none;
  height: 52px;
  padding-top: 16px;
  padding-right: 40px;
  padding-bottom: 16px;
  padding-left: 16px;
  border-radius: 8px;
  border: 1px solid #D3D3E2;
  color: #41414A;
  font-size: 14px;
  font-weight: 400;
}

.c2:focus {
  color: #41414A;
  border-color: #41414A;
}

.c2:focus + .c3 {
  top: 0;
  left: 14px;
  padding-right: 4px;
  padding-left: 4px;
  font-size: 12px;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  -webkit-transition-duration: 500ms;
  transition-duration: 500ms;
  -webkit-transition-timing-function: cubic-bezier(0,0.75,0.1,1);
  transition-timing-function: cubic-bezier(0,0.75,0.1,1);
  font-weight: 700;
  color: #41414A;
}

.c2:focus,
.c2:hover {
  border-color: #41414A;
}

.c2:disabled {
  cursor: not-allowed;
}

.c2[type="number"]::-webkit-outer-spin-button,
.c2[type="number"]::-webkit-inner-spin-button {
  display: none;
}

.c0 {
  position: relative;
  display: inline-block;
  vertical-align: top;
}

.c0 svg {
  position: absolute;
  top: 0;
  right: 0;
  padding-right: 8px;
  padding-left: 8px;
  height: 52px;
  fill: #9898A6;
  outline: none;
  box-sizing: content-box;
  cursor: pointer;
}

.c0 svg:hover,
.c0 svg:focus {
  fill: #41414A;
}

.c0,
.c0 .c1 {
  width: 312px;
}

.c0 .c5 {
  color: #9898A6;
}

<div>
  <div
    class="c0"
  >
    <input
      class="c1 c2"
      type="tel"
      value=""
    />
    <label
      class="c3 c4"
    >
      Label
    </label>
  </div>
</div>
`;
